site/docs/get-started/quickstart.md

```diff
@@ -0,0 +1,81 @@
+# Quick Start Guide
+
+Follow this guide to install Mesh for Data using default parameters that are suitable for experimentation.
+<!-- For a full installation refer to the [full installation guide](./setup/install) instead. -->
+
+## Before you begin
+
+Ensure that you have the following:
+
+- [Helm](https://helm.sh/) 3.3 or newer must be installed and configured on your machine.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as [Kind](http://kind.sigs.k8s.io/) as a cluster administrator.
+
+## Install cert-manager
+
+Mesh for Data requires [cert-manager](https://cert-manager.io) to be installed to your cluster. 
+Many clusters already include cert-manager. Check if `cert-manager` namespace exists in your cluster and only run the following if it doesn't exist:
+
+```bash
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 120s
+``` 
+
+## Install Hashicorp Vault and plugins
+
+[Hashicorp Vault](https://www.vaultproject.io/) and a [secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) plugin are used by Mesh for Data for credential management.
+
+Run the following to install vault and the plugin in development mode:
+
+=== "Kubernetes" 
+
+    ```bash
+    kubectl create namespace m4d-system || true
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/vault-single-cluster/values.yaml \
+        --namespace m4d-system \
+        --wait --timeout 120s
+    ```
+
+=== "OpenShift"
+
+    ```bash
+    kubectl create namespace m4d-system || true
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 \
+        --set "global.openshift=true" \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/vault-single-cluster/values.yaml \
+        --namespace m4d-system \
+        --wait --timeout 120s
+    ```
```

Comment:
@roee88 this part needs to be included in the quick-start...

Reply:
I know :)